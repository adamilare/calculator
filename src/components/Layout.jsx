import { Outlet } from 'react-router-dom';
import NavBar from './NavBar';
import '../css/calculator.css';

const Layout = () => (
  <>
    <NavBar />
    <Outlet />
  </>
);

export default Layout;
