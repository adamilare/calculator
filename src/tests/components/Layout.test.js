import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Layout from '../../components/Layout'; // eslint-disable-line

test('Layout componet renders as expected!', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Layout />
      </MemoryRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
