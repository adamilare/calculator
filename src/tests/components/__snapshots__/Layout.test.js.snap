// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout componet renders as expected! 1`] = `
<div
  className="clas navbar"
>
  <span
    className="nav-left"
  >
    Math Magician
  </span>
  <div
    className="nav-right"
  >
    <a
      aria-current="page"
      className="active"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      className=""
      href="/calculator"
      onClick={[Function]}
    >
      Calculator
    </a>
    <a
      className=""
      href="/quote"
      onClick={[Function]}
    >
      Quote
    </a>
  </div>
</div>
`;
